lib/smart_city/dataset.ex

```diff
@@ -41,7 +41,7 @@ defmodule SmartCity.Dataset do
         }
       ],
       "sourceUrl": "",
-      "protocol": "", //defaults to nil. Can be HTTP1 or HTTP2
+      "protocol": "",       //List of protocols to use. Defaults to nil. Can be [http1, http2]
```

Comment:
can you throw an extra space in here for readability?

Reply:
This though

Reply:
Done.